jobQueue.py

```diff
@@ -152,26 +178,6 @@ def addDead(self, job):
 
         return job.id
 
-    def remove(self, id):
```

Comment:
Although this is not used anywhere currently it would still be useful to keep it, as it can be useful for other extensions (i.e cancelling jobs), so please keep this function

Reply:
Added this back again!